src/nupic/algorithms/TemporalMemory.cpp

```diff
@@ -846,74 +846,11 @@ void TemporalMemory::printParameters(std::ostream& out) const {
       << "connectedPermanence       = " << getConnectedPermanence() << std::endl
       << "minThreshold              = " << getMinThreshold() << std::endl
       << "maxNewSynapseCount        = " << getMaxNewSynapseCount() << std::endl
-      << "checkInputs               = " << checkInputs_ << std::endl
```

Comment:
checkInputs is also technically a parameter, but it's not really useful to know in this table.

checkInputs & extra probably should not be in this table of parameters since they don't really effect the functionality of the TM.  I think that your changes (removals) here should stand.